src/models/config/ChatAnalyticsConfig.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * The main configuration options for Chat Analytics.
+ *
+ * @public
+ */
+export interface ChatAnalyticsConfig {
+  /** The API key for accessing the Analytics Events API. */
+  apiKey: string; // TODO: Add optional "token" for JWT auth
+  /** The environment to send the requests to. */
+  env?: 'PROD' | 'SANDBOX';
```

Comment:
Can we use "PRODUCTION" instead to be consistent with the search config?

Reply:
I suppose the search `experienceVersion` and the Yext `env` are different concepts, but it doesn't hurt to be consistent with the vocab